dashboard/README.md

```diff
@@ -0,0 +1,63 @@
+# Grafana dashboard playground
+
+## Install Docker Desktop
+
+<https://docs.docker.com/desktop/>
+
+## Install Grafana/InfluxDB docker images
+
+```bash
+./install
+```
+
+## Start docker conteiners
+
+```bash
+./start
+```
+
+## Generate InfluxDB random traffic data
+
+### Install luarocks
+
+<https://github.com/luarocks/luarocks/wiki/Download>
+
+### Install luasocket and luafilesystem
+
+```bash
+sudo luarocks install luasocket
+sudo luarocks install luafilesystem
+```
+
+### Start InfluxDB random traffic generation
+
+ ```bash
+cd influx-data-gen
+./influx-data-gen
+```
+
+### Stop InfluxDB random traffic generation
```

Comment:
I would remove this section. It's pretty much implicit that ctrl+c stols execution.

Reply:
I think this is useful for those who are not familiar with the terminal.